src/bsoncxx/json.cpp

```diff
@@ -89,5 +89,11 @@ document::value BSONCXX_CALL from_json(stdx::string_view json) {
     return document::value{buf, length, bson_free_deleter};
 }
 
+document::value BSONCXX_CALL operator"" _bson(const char* str, size_t len) {
+    // len isn't used, but it is required for the UDL to use a char*
```

Comment:
I think you can get around this by changing the signature so it does not name the len variable:
`document::value BSONCXX_CALL operator"" _bson(const char* str, size_t) {...}`

But also, it looks like there should be a signature possible that just takes (const char*) [according to this page](https://en.cppreference.com/w/cpp/language/user_literal), did you have trouble implementing that?

Reply:
I'll try it with `document::value BSONCXX_CALL operator"" _bson(const char* str, size_t) {...}`! Thanks for letting me know you could do that.

I originally tried just using `(const char*)` but it's not an applicable literal operator. On that page, there is a list of accepted parameters and it lists `(const char*)` as being only used:
> as fallbacks for integer and floating-point user-defined literals